.github/CODEOWNERS

```diff
@@ -550,4 +550,10 @@
 /package.json @vojtechjelinek @kevinlee12
 /yarn.lock @vojtechjelinek @kevinlee12
 /scripts/install_third_party_libs.py @vojtechjelinek @kevinlee12
-/.github/ @kevinlee12
+/.github/* @kevinlee12
+
+# Procedure for picking code owners:
+# Codeowners should be selected based on speciality from the following list:
```

Comment:
Please make this more explicit: Codeowners should be one of the team leads from this page:

Also, add to the top of the file and write it as a big NOTE TO DEVELOPERS so that they see it immediately on opening the file.

Reply:
done!